# Topic names
state_topic: "/ur3_cartesian_velocity_controller/ee_state"
command_topic: "/ur3_cartesian_velocity_controller/command_cart_vel"

# Static gain
Kx: 5
Ky: 5
Kz: 5

# Euclidean distance gain
eucl_flag: false
D_eucl: 0.2

# Exponential gain
exp_flag: false
Ka: 10
Kb: 30
min_dis: 0.0
max_dis: 0.2
Ka_exp: 30
Kb_exp: 34
c: 0

